### ISSUE TITLE
Feature Request: Ability to dismiss domain registration notice

### ISSUE THREAD

**Issue by reginabally** [wordpress-mobile/WordPress-Android#14659]
Origin: 3994892-zd-woothemes

Some of the user may have their own domain mapped to the site and do not need to register a domain. It would be nice to be able to dismiss the notice.

**Comment by reginabally** [comment]
Adding the screenshot on WPAndroid 17.3-rc-1

![Screenshot_2021-05-18-17-58-27-26_1cdbe7dded7ec259ed1024b4ff1ae8db](https://user-images.githubusercontent.com/40906847/118632308-dbae2300-b802-11eb-941f-a807871b7715.jpg)

**Comment by thehenrybyrd** [comment]
Initial intent was to request a new feature, such that users could dismiss the notice as desired (regardless of domain status). However, after some discussion and testing @reginabally and I realized that the notice does get removed when a domain is mapped in Android, though not in iOS.
Making the notice dismissible would require that we expose the possibility for domain registration elsewhere, and has not been frequently requested.
Closing this issue for now.
